nodes/src/nodes/cache_llm/IGlobal.py

```
@@ -0,0 +1,118 @@
+
+                    client = redis.Redis(
+                        host=host,
+                        port=port,
+                        db=db,
+                        password=password,
+                        decode_responses=True,
+                        socket_connect_timeout=5,
+                    )
+                    client.ping()
+                    client.close()
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)